docs/installation/autoware/source-installation.md

```diff
@@ -64,7 +64,7 @@ If you want to manually install the dependencies:
 
    ```bash
    source /opt/ros/galactic/setup.bash
-   rosdep install --from-paths src --ignore-src --rosdistro $ROS_DISTRO
+   rosdep install --from-paths src --ignore-src -y --rosdistro $ROS_DISTRO
```

Comment:
[Nits] Writing it first might be easier for users to find it? I feel the `Yes` option should be noticeable.

```suggestion
   rosdep install -y --from-paths src --ignore-src --rosdistro $ROS_DISTRO
```

Reply:
OK